install-kubectl-linux.md: add chmods (#44680)
kubernetes/website: content/en/docs/tasks/tools/install-kubectl-linux.md

   ```shell
   curl -fsSL https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/Release.key | sudo gpg --dearmor -o /etc/apt/keyrings/kubernetes-apt-keyring.gpg

>    sudo chmod 644 /etc/apt/keyrings/kubernetes-apt-keyring.gpg   # Needed so that _apt user can read it

Reviewer: (nit)
```suggestion
   sudo chmod 644 /etc/apt/keyrings/kubernetes-apt-keyring.gpg # allow unprivileged APT tools to read this keyring
```

   ```shell
   # This overwrites any existing configuration in /etc/apt/sources.list.d/kubernetes.list
   echo 'deb [signed-by=/etc/apt/keyrings/kubernetes-apt-keyring.gpg] https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/deb/ /' | sudo tee /etc/apt/sources.list.d/kubernetes.list

>    sudo chmod 644 /etc/apt/sources.list.d/kubernetes.list   # Needed so that command-not-found works correctly

Reviewer: ```suggestion
   sudo chmod 644 /etc/apt/sources.list.d/kubernetes.list   # helps tools such as command-not-found to work correctly
```
